packages/fakta/fakta-omsorg-og-rett/src/opplysningskort/OpplysningerFraSoknad.tsx

```diff
@@ -0,0 +1,185 @@
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import { BodyShort, Label, Table } from '@navikt/ds-react';
+
+import { FaktaKilde } from '@navikt/fp-fakta-felles';
+import { alleKodeverk } from '@navikt/fp-storybook-utils';
+import { type AlleKodeverk, type OmsorgOgRett } from '@navikt/fp-types';
+
+import { EkspansjonsKort } from '../components/ekspansjonsKort/EkspansjonsKort.tsx';
+
+interface Props {
+  omsorgOgRett: OmsorgOgRett;
+  alleKodeverk: AlleKodeverk;
+}
+
+export const OpplysningerFraSoknad = ({ omsorgOgRett }: Props) => {
+  const intl = useIntl();
+
+  const { harRettNorge, harOppholdEØS, harRettEØS, harUføretrygd } = omsorgOgRett.søknad.annenpartRettighet ?? {};
```

Comment:
Kanskje like greit å bruka annenpartRettighet direkte nedover her, for å unngå sjekkar mot både null og undefined